packages/client/builtin/RenderWhen.vue

```diff
@@ -43,6 +47,8 @@ function conditionsMatch(context: Context) {
 <template>
   <div v-if="needsDomWrapper" ref="target">
     <slot v-if="shouldRender" />
+    <slot v-else name="else" />
```

Comment:
Maybe `fallback`? That align with Vue better: https://vuejs.org/guide/built-ins/suspense.html#loading-state